packages/procaptcha/src/modules/Manager.ts

```diff
@@ -74,6 +68,30 @@ const getNetwork = (config: ProcaptchaClientConfigOutput) => {
     return network
 }
 
+const getRandomActiveProvider = (): RandomProvider => {
```

Comment:
Please make a shared function for this in procaptcha-common or something like that